File: leet-code-codebase/0231-power-of-two/0231-power-of-two.py
class Solution(object):
    def isPowerOfTwo(self, n):
        if n == 1:
            return True
        else:
            while(n > 0):
                if n % 2 == 0:
                    if n == 2:
                        return True
                    n = n // 2
                else:
                    return False   
            return False 
        
        

File: leet-code-codebase/arrays/0231-power-of-two/0231-power-of-two.java
class Solution {
    public boolean isPowerOfTwo(int num) {
        if (num == 1){
            return true;
        } else {
            while(num > 0) {
                if (num % 2 == 0) {
                    if (num == 2) {
                        return true;
                    }
                    num = num / 2;
                } else {
                    return false;
                }
            }
            return false;
        }
}
}